runtime/peaq-dev/src/lib.rs

```diff
@@ -859,7 +864,7 @@ macro_rules! impl_to_pot_adapter {
 	};
 }
 
-impl_to_pot_adapter!(ToStakingPot, PotStakeId, NegativeImbalance);
+// impl_to_pot_adapter!(ToStakingPot, PotStakeId, NegativeImbalance);
 impl_to_pot_adapter!(ToMachinePot, PotMorId, NegativeImbalance);
```

Comment:
Do you think it's better to move the on_unbalanced to the MorPallet in the future?

Reply:
Yes :+1: in my opinion these are machanisms that do belong to the pallets. The pallets "want" to be used like that, so we should integrate it within the pallets :+1: